ls1intum/Artemis
# `Programming exercises`: Fix the tooltip placement on the programming exercise timeline

### Checklist
#### General



#### Client

### Motivation and Context


The tooltips for the Timeline when creating a new Programming Exercise were sometimes displayed to the bottom and sometimes to the top. Also the text was only shown partially.

### Description

I fixed the issue https://github.com/ls1intum/Artemis/issues/5125.

### Steps for Testing

Prerequisites:
- 1 Course

1. Go to course
2. Click "Generate new Programming Exercise"
3. Scroll to Timeline
4. Check if the tooltips are now displayed correctly
5. Verify, that no other tooltip is broken now

### Review Progress





#### Code Review
#### Manual Tests

### Test Coverage



_unchanged_

### Screenshots


![video](https://user-images.githubusercontent.com/44734881/183684736-f13cb9ba-d9b0-4500-809a-cef40d53c4b7.gif)
